packages/admin/src/config/_mixin.tsx

```diff
@@ -0,0 +1,23 @@
+// sizes for media queries
+const sizes = {
```

Comment:
밑에는 fontSize인데 먼가 안맞는것 같아서
```suggestion
const deviceSize = {
```